### PR TITLE
tweak: update guides to reflect npm support

### PR DESCRIPTION
**Pull request by roperzh**

### CODE REVIEW

File: docs/getting-started/sketch.md [hunk 1]
```diff
@@ -46,9 +46,14 @@ Mark any visuals you want to use in your Diez project for export (or create slic
...
 Make sure to save your changes. Now you're all set up to sync your Sketch file with your Diez project. Head back over to your terminal and within your Diez project folder run:
 
-```bash
-yarn diez extract
+<CodeTabs>
+```bash tabname-yarn
+$ yarn diez extract
 ```
+```bash tabname-npm
+$ npx diez extract
```
Comment (taylorpoe) [inline docs/getting-started/sketch.md]: @roperzh Just curious why this command is npx & not npm?

**Comment by taylorpoe** [comment]
LGTM

File: docs/getting-started/sketch.md (hunk 1 above)
Comment (roperzh) [reply docs/getting-started/sketch.md]: In this particular case, `npx` is they npm-way to run a locally installed binary (in this case the repo copy of `diez`), for this purposes it is the equivalent of running `yarn diez ...`.

However, if you want to run a script in your `package.json` for example, `npm run script-name` is the way to go.

**Comment by roperzh** [comment]
Thanks! merging.

**Pull request merged**